_summaries/image-png.md

```diff
@@ -0,0 +1,5 @@
+---
+name: "png"
+---
+
+Decoder and Encoder for the wide spread `png` format.
```

Comment:
Sorry about the nitpicking, but it is nice to fix stuff like this. Shouldn't `encoder` we be written with a small "e" here, and I think widespread is one word.

Reply:
All good, fix all the nits!